realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -91,11 +93,10 @@ BaseRealSenseNode::BaseRealSenseNode(rclcpp::Node& node,
     {
         ROS_INFO("Intra-Process communication enabled");
     }
-    else
-    {
-        // intra-process requirment of QoS.durability=Volatile cannot be fulfilled with `StaticTransformBroadcaster` as it only support `TransientLocal` durability.
-        _static_tf_broadcaster = std::make_shared<tf2_ros::StaticTransformBroadcaster>(node);
-    }
+
```

Comment:
Can you please add a special case for dashing which doesn't support `tf2_ros::StaticBroadcasterQoS()`?